brainprep/workflow/prequal.py

```diff
@@ -47,47 +48,126 @@ def brainprep_prequal(dwi, bvec, bval, pe, readout_time, output_dir, t1=None):
     In order to use the synb0 feature you must bind your freesurfer license as
     such: -B /path/to/freesurfer/license.txt:/APPS/freesurfer/license.txt
     """
-    print_title("PreQual dtiQA pipeline")
-    if pe in ["i", "j", "k"]:
-        pe_axis = pe
-        pe_signe = "+"
-    elif pe in ["i-", "j-", "k-"]:
-        pe_axis = pe[0]
-        pe_signe = pe[1]
+    if len(dwi.split(",")) == 2 and len(bvec.split(",")) == 2 and\
```

Comment:
Write a utility function that unpacks a list written as a string separated by comas:

```
  def listify(obj):
    if not isinstance(obj, list):
       obj = obj.split(",")
    return obj
```

Apply it to the whole PR.